### DISCUSSION TITLE
default apps

### DISCUSSION THREAD

**Discussion by AuracleTech** [linuxmint/discussions#15] [Ideas]
There could be an option to download the OS without any apps installed
Because I'm gonna be honest I am never going to be using these
Windows is installed already bloated but this also feels like it, I wish it could clean install
![image](https://github.com/linuxmint/discussions/assets/40531871/63c89be1-fd22-4227-b1ad-1e77cbb73834)

**Comment by clefebvre** [comment]
It's designed with most people in mind. Some people need more apps, some less. We can't make an option for each different niche audience. What we do though is make it easy to add and/or remove software.

**Reply by AuracleTech** [reply]
Even windows doesn't come with that much bloat . . .
